schemas/test/en/test_calculated_summary.json

```diff
@@ -339,8 +381,18 @@
                             "id": "percentage-total-playback",
                             "title": "We calculate the total of percentage values entered to be %(total)s. Is this correct?",
                             "calculation": {
-                                "calculation_type": "sum",
-                                "answers_to_calculate": ["fifth-percent-answer", "sixth-percent-answer"],
+                                "operation": {
```

Comment:
Not sure I was quite clear in the last comment I made, but the removal of the old when rules is separate to calculated summary. For now we still need to support calculated summary in both the old and new format, so some of these schemas and tests will need to be reverted.

Reply:
Brought back the old calculated summary schemas whilst updating their when rules